### ISSUE TITLE
escape sql variables, not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1822]
```abap
SELECT cimstp segtyp INTO (ls_idocsyn-segtyp, ls_idocsyn-parseg) FROM cimsyn WHERE cimtyp = bar ORDER BY nr DESCENDING.
ENDSELECT.
```
